kubeflow/tf-serving/tf-serving.libsonnet

```diff
@@ -147,6 +147,13 @@
         runAsUser: 1000,
         fsGroup: 1000,
       },
+      volumeMounts+: if 'modelStorageType' in $.params then
```

Comment:
do you need this "if 'modelStorageType' in $.params then"? It seems redundant?
I think you can add modelStorageType into param and default to cloud or empty string.